google/cloud/security/common/data_access/sql_queries/create_tables.py

```diff
@@ -20,8 +20,8 @@
         `project_number` bigint(20) NOT NULL,
         `project_id` varchar(255) NOT NULL,
         `project_name` varchar(255) DEFAULT NULL,
-        `lifecycle_state` enum('ACTIVE','DELETE_REQUESTED',
-            'DELETE_IN_PROGRESS','DELETED') DEFAULT NULL,
+        `lifecycle_state` enum('LIFECYCLE_STATE_UNSPECIFIED','ACTIVE',
```

Comment:
Will this suffice? https://github.com/GoogleCloudPlatform/forseti-security/blob/master/google/cloud/security/common/gcp_type/resource.py#L59